spacy_llm/tasks/ner.py

```diff
@@ -99,19 +99,24 @@ def _unique(items: Iterable[str]) -> Iterable[str]:
 @registry.llm_tasks("spacy.NER.v1")
 def make_ner_task(
     labels: str,
-    template: str = _DEFAULT_NER_TEMPLATE,
+    template: TemplateConfigType = _DEFAULT_NER_TEMPLATE,
     examples: ExamplesConfigType = None,
     normalizer: Optional[Callable[[str], str]] = None,
     alignment_mode: Literal["strict", "contract", "expand"] = "contract",  # noqa: F821
     case_sensitive_matching: bool = False,
     single_match: bool = False,
 ) -> "NERTask":
     labels_list = split_labels(labels)
+    # Load examples
     raw_examples = examples() if callable(examples) else examples
     ner_examples = [NERExample(**eg) for eg in raw_examples] if raw_examples else None
+    # Load templates
+    if template is None:
+        raise ValueError("A template must be supplied. It cannot be 'None'.")
```

Comment:
Decided to just raise an error when `template` is `None` rather than assigning default. Not too attached with this decision though. Happy to hear alternatives!